src/views/party-settings.ts

```diff
@@ -127,6 +130,14 @@ const PartySettings = (props: PartySettingsProps & PartySettingsDispatch) => htm
             Display "Ken Burns" background in TV mode
         </paper-checkbox>
 
+        <br />
+
+        <paper-checkbox checked="${props.syncedPlayback}"
+                        on-checked-changed="${ev => props.syncPlayback((ev.target as HTMLInputElement).checked)}"
+                        title="Syncs your playback with the party host.">
```

Comment:
I've only just realised that this setting is only seen by the host so the wording here seems odd... any suggestions? Maybe "sync your playback with the listeners"?

Reply:
Umm, this might need a fundamentally different approach, UX wise. Maybe we should enable the play button for parties where a host is playing, so that other people can start their instance of Festify in slave mode? Currently the button is disabled for everyone who doesn't have admin access to the party / is host.

This would require quite some refactoring in player handling, though.

Reply:
Yeah, that was my concern as the UX doesn't feel right to be honest.